packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -141,19 +142,15 @@ const InlineMediaIndicator = styled.div`
 export const Headline = styled.h3`
   ${props => (props.script ? getPica(props.script) : '')};
 
-  ${({ script, topStory }) => {
-    if (!script) {
-      return '';
-    }
+  ${({ script, topStory }) => (script && topStory ? getParagon(script) : '')}}
```

Comment:
This doesn't look right, should this line be merged with the one above?

Reply:
I think the lines you deleted on the left hand side of this diff were good, just the `getPica` above needed to be deleted

Reply:
Applied some changes, but am not sure if i got your suggestion right.